### PR TITLE
update the error test

### PR DESCRIPTION
**Pull request by huangyiminghappy**
infact want to test the waittask

### CODE REVIEW

**Comment by huangyiminghappy** [comment]
update

File: conductor.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cyzhao) [inline conductor.iml]: this file should not checked in

**Comment by v1r3n** [comment]
@huangyiminghappy can you send the PR against `dev` branch?

**Comment by v1r3n** [comment]
@huangyiminghappy can you remove the `conductor.iml` and send the PR against `dev` branch?

**Comment by huangyiminghappy** [comment]
thanks

**Pull request closed without merging**